format/index.md

```diff
@@ -527,6 +529,28 @@ the value of each member of a `links` object is not constrained by the
 current version of the spec. It should be assumed that anything can be
 allowed in the future for such values: object, array, or scalar.
 
+### JSON API Object <a href="#document-structure-jsonapi-object" id="document-structure-jsonapi-object" class="headerlink"></a>
+
+A JSON API document **MAY** include information about its implementation under
+a top level `"jsonapi"` member. If present, it **MUST** be an object containing
+a `version` member whose value is a string indicating the highest JSON API
+version supported. Other than `"meta"`, servers **MUST NOT** include any
```

Comment:
@dgeb Should this read: "a `version` member... indicating the highest JSON API version at least partially supported"?Because I'm imagining that a server could support some optional features from 1.1 without fully supporting 1.0's optional features, and the reference to "partial support" would seem to make it less clearer that the server can return 1.1 even though it hasn't completely implemented 1.0.